README.md

```diff
@@ -51,7 +51,8 @@ Download from our Github releases [page](https://github.com/sedaprotocol/seda-ch
 
 ### Build From Source
 
-Please check out [developing](DEVELOPING.md).
+Note that the `main` branch is unstable. For a stable version, `git checkout` a tag that has been released.
```

Comment:
Just a minor suggestion, but you can leave as it is:
```suggestion
Note that the `main` branch is under development and might not always be stable. For a stable version, consider checking out a released tag using `git checkout`.
```

Reply:
perfect. updated